README.md

```diff
@@ -334,7 +334,7 @@ These options are available when running with `--long` (`-l`):
 
 Some of the options accept parameters:
 
-- Valid **--color** options are **always**, **automatic**, and **never**.
+- Valid **--color** options are **always**, **automatic** (or **auto** for short), and **never**.
```

Comment:
```suggestion
- Valid **--co{u}lor** options are **always**, **automatic** (or **auto** for short), and **never**.
```

Reply:
Sorry for not followinig what are already presented in `README.md`.  
We have the following lines writing `colo[u]r`. Definitely should conform to this.

```markdown
- **-F**, **--classify**: display type indicator by file names
- **--colo[u]r**: when to use terminal colours
```

Reply:
it is okay everyone make mistakes